cityehr-quick-start-guide/src/main/xslt/create-topic-html.xslt

```diff
@@ -25,15 +34,29 @@
         </xsl:call-template>
       </head>
       <body>
-        <div id="top-nav">
+        <nav id="top-nav">
           <xsl:apply-templates select="." mode="top-nav"/>
-        </div>
+        </nav>
+
+        <!-- Petal Edit Button -->
+        <section>
```

Comment:
As I understand the use of `section` tag in HTML 5, it shouldn't really be used for this purpose. Should this go in the `nav`, or perhaps a separate `div` with id would be more appropriate?

Reply:
@adamretter I see your point, and the `section` element is not the best choice here. 
The button is not part of a navigation. The `nav` is already represented by your navigation element on top and at the end of the page.
The best practice for choosing a `div` is to use it only for layout reasons, because it has no semantical meaning.
Therefore I went for the `section`, even though it is a stand-alone button at the moment, but I hoped we could add some information next to it, perhaps?
I could use an `aside` element, that represents supplementary content which is not part of the main document. How about that?

Reply:
@plutonik-a As I understand the use of section tag in HTML 5, an `aside` isn't suitable here. Please change this to a `div` with an appropriate `id`.

Reply:
Fixed with a3b2751